models/migrations/v161.go

```diff
@@ -22,6 +22,7 @@ func convertTaskTypeToString(x *xorm.Engine) error {
 		FEISHU
 		MATRIX
 		WECHATWORK
+		PACKAGIST
```

Comment:
Here and below and v162.go, the `PACKAGIST` doesn't need to be processed. At that time we did not have it yet.

Reply:
Thanks for your review and the approval. I've removed the migrations from v161.go and v162.go.

I had already suspected that those migrations are not necessary, but I've looked at the PR for the last webhook addition (wechatnetwork, #15910). In that PR, those migrations have been added. AFAIU, the migrations for WeChat were not necessary either, right?

I've also merged in the latest changes from the gitea main branch and squashed the commits, to keep the commit for this PR as minimal as possible. If you'd like to have separate commits, please say so, and I'll force-push again.

Reply:
Oh yep, adding or not are both fine .... just small nit, nvm. I didn't review the wechatnetwork PR  #15910  😂